scripts/wsrep_sst_common.sh

```diff
@@ -287,15 +287,24 @@ wsrep_log_debug()
 #
 get_mysqld_path()
 {
+
     # Locate mysqld
     # mysqld (depending on the distro) may be in a different
     # directory from the SST scripts (it may be in /usr/sbin not /usr/bin)
     # So, first, try to use readlink to read from /proc/<pid>/exe
     if which readlink >/dev/null; then
+
```

Comment:
If `readlink` returns with error, are we sure that `MYSQLD_PATH` is still empty?
Maybe we should check the readlink error?

Reply:
Done